File: day6/app/src/main/java/fan/akua/day6/fragments/MineFragment.java
package fan.akua.day6.fragments;

import androidx.fragment.app.Fragment;

import fan.akua.day6.R;

public class MineFragment extends Fragment {
    public static MineFragment newInstance() {
        return new MineFragment();
    }

    private MineFragment() {
        super(R.layout.fragment_mine);
    }
}

File: day11/app/src/main/kotlin/fan/akua/day11/fragments/MineFragment.kt
package fan.akua.day11.fragments

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.View
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.fragment.app.Fragment
import fan.akua.day11.R


class MineFragment : Fragment(R.layout.fragment_mine) {
    private lateinit var webView: WebView

    @SuppressLint("SetJavaScriptEnabled")
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        webView = view.findViewById(R.id.webview)
        webView.setWebViewClient(WebViewClient())
        webView.getSettings().javaScriptEnabled = true
        webView.loadUrl("http://blog.akua.fan")
    }

    override fun onDestroyView() {
        webView.destroy()
        super.onDestroyView()
    }
}